app/javascript/components/layout/CustomTable.js

```diff
@@ -138,7 +138,7 @@ class CustomTable extends React.Component {
             <Spinner variant="secondary" animation="border" size="lg" />
           </div>
         )}
-        <Table striped bordered hover size="sm">
+        <Table responsive striped bordered hover size="sm">
```

Comment:
Since all of our tables (that use this class) would inherit this, are we okay with this change globally?

Reply:
Yes, I believe this actually addresses the overflow issue described in this ticket: [SARAALERT-1090](https://tracker.codev.mitre.org/browse/SARAALERT-1090). Will do more testing to verify as well.